pmd-core/src/main/java/net/sourceforge/pmd/util/StringUtil.java

```diff
@@ -283,6 +261,69 @@ public static void appendXmlEscaped(StringBuilder buf, String src, boolean suppo
         }
     }
 
+
+    /**
+     * Replace some whitespace characters so they are visually apparent.
+     *
+     * @param o
+     *
+     * @return String
+     */
+    public static String escapeWhitespace(Object o) {
```

Comment:
Right I sometimes hit the "format" shortcut mechanically, which is fine when I refactor an entire class but not so much for small changes x) I've now disabled code rearrangement